AegeanTools/CLI/AeReg.py

```diff
@@ -53,7 +53,7 @@ def main(argv=()):
     invocation_string = " ".join(argv)
 
     # configure logging
-    logging_level = logging.DEBUG if options.debug else logging.INFO
+    logging_level = logger.debug if options.debug else logger.info
```

Comment:
This was correct in the initial version.

Reply:
Yes,  it was. Another victim of a global search and replace.